custom_components/tahoma/tahoma_api.py

```diff
@@ -629,19 +634,24 @@ def label(self):
 
     @property
     def command_definitions(self):
-        """List of command devinitions."""
-        return self.__definitions["commands"]
+        """List of command definitions."""
+        return self.__command_def
 
     @property
     def state_definitions(self):
-        """State of command devinition."""
-        return self.__definitions["states"]
+        """List of state definition."""
+        return self.__states_def
 
     @property
     def active_states(self):
         """Get active states."""
         return self.__active_states
 
+    @property
+    def attributes(self):
```

Comment:
Good to have those present! Would be good to understand if we can make the component more dynamic by looking at the attributes, for example for covers and sensors.